src/Aquifer.API/Endpoints/Resources/ParentResources/List/Response.cs

```diff
@@ -10,6 +10,7 @@ public class Response
     public required bool Enabled { get; set; }
     public required string ShortName { get; set; }
     public required string DisplayName { get; set; }
+    public required string Code { get; set; }
```

Comment:
Is this leftover or still being used on the front-end somewhere? Didn't see it getting sent back in any of the new endpoints.